Makefile

```diff
@@ -1,22 +1,23 @@
 VENV_NAME=.venv
 PYTHON=${VENV_NAME}/bin/python3
 PIP=${VENV_NAME}/bin/pip
+COMMON_VARS ?= PYDEVD_DISABLE_FILE_VALIDATION=1 JUPYTER_PLATFORM_DIRS=1
 
 SUBSTRATUS_BRANCH ?= main
+PROJECT_ID ?= substratus-integration-tests
```

Comment:
I can address this in follow up PR if you prefer to merge this first. We can set the default project in the github action itself. I think setting project ID doesn't belong in the makefile which is something everyone would use. `make test` should use the current default project that's configured.

Reply:
Looks like you already set the project ID: https://github.com/substratusai/substratusai.github.io/pull/31/files#diff-2154d5cafbed57574ae09e3c4689fb9aa37e3472f599d6a27c00e5564149f4e6R37

so this shouldn't be needed in the Makefile?

Reply:
At the makefile level yeah it should probably be left off. I removed it and it all seems to work still